src/api/users.js

```diff
@@ -24,7 +25,8 @@ export async function authenticate (ctx, email) {
   email = unescape(email)
 
   try {
-    const user = await UserModelAPI.findOne({email}).exec()
+    const emailReg = new RegExp(`^${_.escapeRegExp(email)}$`, 'i')
```

Comment:
Perhaps `new RegExp(`^${_.escapeRegExp(email)}$`, 'i')` should be moved into a separate function instead of being repeated.